src/Neo.CLI/Settings.cs

```diff
@@ -44,25 +44,30 @@ public static Settings Default
                     var config = new ConfigurationBuilder().AddJsonFile("config.json", optional: true).Build();
                     Initialize(config);
                 }
-
-                return s_default!;
+                return Custom ?? s_default!;
             }
         }
 
+        public static Settings? Custom { get; set; }
```

Comment:
Set `s_default`?

Reply:
no, its not a property.

Reply:
he means it a property and no need to change it @shargon

Reply:
> he means it a property and no need to change it @shargon

LOL, u r right